templates/homepage.html

```diff
@@ -34,11 +34,23 @@
 </head>
 
 <body>
+    <div id="user_info">
```

Comment:
This looks like this will eventually behave as a navigation bar. Here's a helpful document https://www.w3schools.com/howto/howto_js_topnav.asp